entrypoint.sh

```diff
@@ -43,7 +43,6 @@ while true ; do
     "Authorization: Bearer $FLYWAY_HUB_ACCESS_TOKEN")
 
   status=$(echo $result | jq -r '.status')
-  echo "migration test status: $status"
```

Comment:
Is there like a debug logging for GH actions that we would use so people know it's still working?

Reply:
There isn't anything like that I can find. The way to do this IMO would be to add a `logLevel` argument to the action that could allow for more verbose logging including these progress updates.